### PR TITLE
CTF Java HackFest 2015

### PR DESCRIPTION
**Pull request by MrDoel**

### CODE REVIEW

**Comment by dhanvi** [comment]
@YASME-Tim can you check this again I think this doesn't look good to be :-1: 

Seems like I have messed up little bit :disappointed:

**Comment by MrDoel** [comment]
Sorry, i removed it, there is a little wrong on my CTF. Will update soon

File: ctf-javahackfest-2015/begin/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Point: 50
+
+Pada soal pertama ini, panitia hanya memberikan sebuah file zip dan dalam file
+tersebut terdapat flag "StartJavaHackFest2015"
```
Comment (dhanvi) [inline ctf-javahackfest-2015/begin/README.md]: also better make the challenges in English, I think @abdilahrf can help you with these :smile:

**Comment by MrDoel** [comment]
I can , ok :)

**Comment by YASME-Tim** [comment]
@MrDoel What kind of CTF was this? 5 participants? Please provide all descriptions and create a ctf directory with [this tool](https://github.com/ctfs/write-ups-tools/tree/master/genctf). @dhanvi You should not have accepted this pull request. 23 commits for 10 empty files and 2 README.mds in Indonesian.

**Comment by dhanvi** [comment]
@YASME-Tim  Ya I shouldn't have accepted the pull request, It was my mistake for the first time when I saw it I felt it was good but soon found that it isn't that good so I reverted the pull

File: ctf-javahackfest-2015/begin/README.md (hunk 1 above)
Comment (abdilahrf) [inline ctf-javahackfest-2015/begin/README.md]: he is can translate to english :)

**Pull request merged**